src/main/resources/roleconfig/role_common.json

```diff
@@ -1723,13 +1725,230 @@
         },
         "attributes": {
           "jurisdiction": {
+            "mandatory": true,
+            "values": ["PUBLICLAW"]
```

Comment:
no need to add jurisdiction to special access cases. please remove "values": ["PUBLICLAW"]